src/main/java/de/craften/ui/swingmaterial/Utils.java

```diff
@@ -0,0 +1,185 @@
+                }
+            }
+        } catch (ClassNotFoundException ex) {
+            //It seems not
+            found = false;
+        }
+        useSun2D = found;
+        getUsableBounds = getMethod;
+    }
+    
+    /**
+     * Checks the area available in the desktop, excluding the taskbar.
+     * In order to do this, an attempt to call
+     * <code>sun.java2d.SunGraphicsEnvironment.getUsableBounds()</code> is
+     * performed. If this can't be done, the method falls back to the default
+     * <code>Toolkit.getDefaultToolkit().getScreenSize()</code>, although such
+     * method doesn't exclude the taskbar area.
+     * @return A Rectangle with the usable area for maximized windows.
+     * @author DragShot
+     */
+    public static Rectangle getScreenSize(){
+        Rectangle screen;
+        if (useSun2D) { //Use sun.java2d.SunGraphicsEnvironment.getUsableBounds()
+            try {
+                Frame frame = new Frame();
+                GraphicsConfiguration config = frame.getGraphicsConfiguration();
+                screen = (Rectangle)getUsableBounds.invoke(null, config.getDevice());
+                frame.dispose();
+            } catch (Exception ex) {
+                //If something doesn't work, fallback to Toolkit
+                Dimension size = Toolkit.getDefaultToolkit().getScreenSize();
+                screen = new Rectangle(0, 0, size.width, size.height);
+            }
+        } else { //Do it the traditional way
+            Dimension size = Toolkit.getDefaultToolkit().getScreenSize();
+            screen = new Rectangle(0, 0, size.width, size.height);
+        }
+        return screen;
+    }
+    
+    /**
+     * Checks if the translucency effect is supported. Java 6 does not support
+     * this. Only Java 7 and higher VMs might do, depending of the Graphics
+     * Environment and OS.
+     * @return <code>true</code> if translucency is supported,
+     *         <code>false</code> otherwise.
+     * @author DragShot
+     */
+    public static boolean isTranslucencySupported(){
+        boolean nativeTrans;
+        if (System.getProperty("java.version").contains("1.6")) {
+            System.err.println("Per-pixel translucency is currently not "
+                    + "supported.\nPlease upgrade your JRE to at least Java 7 "
+                    + "to support this feature.");
+            nativeTrans = false;
+        } else {
+            nativeTrans = GraphicsEnvironment
+                    .getLocalGraphicsEnvironment().getDefaultScreenDevice()
+                    .isWindowTranslucencySupported(GraphicsDevice
+                    .WindowTranslucency.PERPIXEL_TRANSLUCENT);
+        }
+        return nativeTrans;
+    }
+    
+    /**
+     * Determines if a given {@link Color} is dark enough for white text to be
+     * seen more easily than black text. This tries to stick to the Material
+     * Color Guide as much as possible, and although two or three of the color
```

Comment:
Instead of writing a "magic" formula, we could just hard-code that for the `MaterialColor`s and add a `isDark()` method to the enum.
Or at least, add that method to the enum, it might be useful. :wink: